src/commands/init.command.ts

```diff
@@ -122,6 +122,7 @@ const copyTemplate = async ({
 }) => {
   const spinner = ora('Creating workspace').start();
   await fs.copy(template.path, workspace.path);
+  await renameGitignoreFiles(workspace.path);
```

Comment:
This is also how `create-analog` handles it https://github.com/analogjs/analog/blob/4f3427b0ae0b4ee0167a61e92b1be1c29ec4366c/packages/create-analog/index.js#L46